provisioning/device/readme.md

```diff
@@ -1,13 +1,22 @@
-#azure-iot-provisioning-device
-Device client to Communicate with Azure Device Provisioning Service
+This package (**azure-iot-provisioning-device**) provides the device client that is used to communicate with the Azure IoT Hub Device Provisioning Service.
```

Comment:
> Hub  [](start = 122, length = 5)

We are really using the word "hub" prior to the DPS part?

Reply:
yup

---
In reply to: [157664937](https://github.com/Azure/azure-iot-sdk-node/pull/184#discussion_r157664937) [](ancestors = 157664937)